corehq/apps/userreports/views.py

```diff
@@ -1127,7 +1128,20 @@ def post(self, request, *args, **kwargs):
     def get_reports(self):
         reports = StaticReportConfiguration.by_domain(self.domain)
         reports += ReportConfiguration.by_domain(self.domain)
-        return [report for report in reports if report.table_id == self.config.table_id]
+        ret = []
+        for report in reports:
+            try:
+                if report.table_id == self.config.table_id:
+                    ret.append(report)
+            except DataSourceConfigurationNotFoundError:
+                _soft_assert = soft_assert(to=[
+                    '{}@{}'.format(name, 'dimagi.com')
+                    for name in ['jemord', 'cellowitz', 'npellegrino', 'frener']
```

Comment:
I'm sending to you guys as well so thatyou can handle the enikshay errors. I'll take care of the icds ones and can own it indefinitely